core/src/main/java/com/cloud/agent/api/storage/MigrateVolumeCommand.java

```diff
@@ -36,6 +36,7 @@ public class MigrateVolumeCommand extends Command {
     String attachedVmName;
     Volume.Type volumeType;
     String hostGuidInTargetCluster;
+    Long newIops;
```

Comment:
Can you explain why newIops is added , not newIopsRead/newIopsWrite ?

Reply:
VMware hypervisor does not allow specifying the IOPS for read and write operations. Instead, you can only specify the IOPS for the disk.

![image](https://user-images.githubusercontent.com/31869303/220373266-8b05c381-c2dd-4319-84c8-324da245bfc7.png)

Reply:
thanks @SadiJr 
MigrateVolumeCommand is a class in core module. It might be used if there are similar issues with kvm and/or xenserver which support IOPS read/write.